### DISCUSSION TITLE
How to reference data from other panels?

### DISCUSSION THREAD

**Discussion by jpyun0625** [gapitio/gapit-htmlgraphics-panel#82] [Q&A] [answered]
Hello
Can I use data from other panels (timeseries, table, etc...) in htmlgraphics panel?
ex) Display legend of graph panel in htmlgraphics

If possible, what should I refer to as console.log() in onRenderJS to check?

**Comment by flesa** [comment, marked as answer]
Hello

You could try using the native [-- Dashboard --](https://grafana.com/docs/grafana/latest/panels/share-query-results/) data source.

This should let you find all query data from the selected panel using console.log(data).
